examples/test_scr_poststage

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# This script generates checkpoints on the IBM burst buffer disk, and then
+# starts transferring them to GPFS before exiting.  Then, the scr_poststage
+# script gets run after the transfers finish, and finalizes them.
+#
+# Example:
+#
+# # Allocate one node with 100GB burst buffer, run test_scr_poststag to
+# # create and begin transferring checkpoints, and after the transfers finish
+# # (after the job ends), call scr_poststage to finalize them.
+# bsub -nnodes 1 -stage storage=100:out=,scr_poststage lexec jsrun -r 1 test_scr_poststage
+#
+
+# Where do you want to write your checkpoints to?  This should be somewhere in
+# on the burst buffer filesystem.
+
+BB_DIR="$BBPATH/my_checkpoints"
+
+# Where do you want your checkpoints to transfer to in GPFS?
+GPFS_DIR="/p/gpfs1/$(whoami)"
+
+if [ -z "$BBPATH" ] ; then
+    echo "BBPATH isn't set in the script, please set it"
+    exit
+fi
+
+if [ -z "$GPFS_DIR" ] ; then
+    echo "Please set \$GPFS_DIR in test_scr_poststage"
+    exit
+fi
+
+# Path to SCR's 'test_api' and 'test_multiple' binaries
+# PATH="$PATH:~/scr_poststage/build/examples/"
+
+if ! which test_api &> /dev/null ; then
+    echo "Please add the path to SCR's 'test_api' binary into your \$PATH or into test_scr_poststage itself"
+    exit
+fi
+
+function make_scr_conf {
+    echo "
+SCR_COPY_TYPE=FILE
+
+SCR_FLUSH=1
+SCR_FLUSH_ASYNC=1
+SCR_FLUSH_POSTSTAGE=1
+SCR_CACHE_BYPASS=0
+
+# Make our burst buffer a cache device
+STORE=$BB_DIR GROUP=NODE COUNT=1 FLUSH=BBAPI
+
+CKPT=0 INTERVAL=1 GROUP=NODE   STORE=$BB_DIR TYPE=PARTNER
+
+SCR_CNTL_BASE=$BB_DIR
+SCR_DEBUG=10"
+}
+
+make_scr_conf > /tmp/test_scr_poststage.config
```

Comment:
Let's move the config file to GPFS since that's more in line with what we recommend to users.  This line will then need to come after the ``mkdir -p $GPFS_DIR`` line with that switch.
```suggestion
make_scr_conf > /$GPFS_DIR/test_scr_poststage.config
```